packages/dds/merge-tree/src/snapshotLoader.ts

```diff
@@ -154,13 +154,13 @@ export class SnapshotLoader {
     }
 
     private async loadBody(chunk1: MergeTreeChunkV1, services: IChannelStorageService): Promise<void> {
-        this.runtime.logger.shipAssert(
+        assert(
```

Comment:
i actually expected this to throw. a little scary to find out its not

Reply:
Do  you know where the thrown error would be caught and if it's logged at that point? Now that it's not being logged here.

Reply:
Likewise for all these changes...

Reply:
we throw in other places in the snapshot load flow, and those are caught and handled correctly. no different here